src/Microsoft.Windows.CsWin32/templates/BOOL.cs

```diff
@@ -1,6 +1,11 @@
 ﻿partial struct BOOL
 {
+	public const int Size = sizeof(int);
+	public static BOOL TRUE { get; } = new(true);
+	public static BOOL FALSE { get; } = new(false);
```

Comment:
Its used a lot in XML comments and tests. I'd say the code that does use it should really just use `bool` and the conversion.

Reply:
Thanks. That sounds like legacy that could be changed rather than some functional justification. I'll suggest you either update the comments and tests or define your own partial class for this.